common/persistence/cassandraPersistence_test.go

```diff
@@ -73,11 +73,11 @@ func (s *cassandraPersistenceSuite) TestPersistenceStartWorkflow() {
 		WorkflowId: common.StringPtr("start-workflow-test"),
```

Comment:
Can you also update existing test to assert workflow_timeout after reading from DB matches the expectation.
Also have a test after to validate value after update.

Reply:
Done